app/src/stylesheets/content/main.sass

```
@@ -0,0 +1,104 @@
+.main-content
+    padding: 1rem 10%
+.main_content_more-button
+    color: #fa5c65
+    font-family: Montserrat
+    font-size: 1.25rem
+    font-weight: 400
+    line-height: 20px
+    background: transparent
+    padding: .5rem 0
+    text-transform: uppercase
+.main-content_advantages
+    text-align: left
+    @extend .d-flex !optional
+    @extend .flex-wrap !optional
+    @extend .justify-content-around !optional
+    .main-content_advantages_article
+        width: 30%
+        margin: auto auto
+    .main-content_advantages_more-button
+        @extend .main_content_more-button !optional
+    .main-content_advantages_header
+        color: #465065
+        font-family: Montserrat
+        font-size: 1.25rem
+        font-weight: 700
+        line-height: 1.5
+        text-transform: uppercase
+    .main-content_advantages_description
+        color: #9aa0ab
+        font-family: "Open Sans"
+        font-size: 1rem
+        font-weight: 400
+        line-height: 1.5
+.main-content_portfolio-section_portfolio
+    text-align: left
+    @extend .d-flex !optional
+    @extend .flex-wrap !optional
+    @extend .justify-content-around !optional
+    .main-content_portfolio-section_portfolio_header
+        color: #314a5b
+        font-family: Montserrat
+        font-size: 2.25rem
+        font-weight: 700
+        line-height: 1.5
+        text-transform: uppercase
+    .main-content_portfolio-section_portfolio_slogan
+        font-family: Montserrat
```

Не забывай указывать дефолтный font-family. Напирмер `sans-serif`
```suggestion
        font-family: Montestrat, sans-serif
```